### PR TITLE
log: Use error log type instead of Application/MinIO type

### PR DESCRIPTION
**Pull request by vadmeste**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Introduce new logs levels. This change is minimal to include the new type logs

## Motivation and Context
New log levels

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Comment by klauspost** [comment]
Yeah. Never understood what the "kind" was anyway.

**Comment by harshavardhana** [comment]
> Yeah. Never understood what the "kind" was anyway.

It wasn't very kind on the eyes.

File: cmd/test-utils_test.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func TestMain(m *testing.M) {
 	// Disable printing console messages during tests.
 	color.Output = io.Discard
 	// Minimum is error logs for testing
```
Comment (donatello) [inline cmd/test-utils_test.go]:
```suggestion
	// Disable Error logging in testing.
```

**Review by donatello — APPROVED** [review]
Small comment fix suggested. Otherwise LGTM

File: cmd/test-utils_test.go (hunk 1 above)
Comment (donatello) [reply cmd/test-utils_test.go]: Just fixes out-of-date comment.

**Pull request merged**